extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -1428,6 +1452,7 @@ export const popupMessageListener = (request: Request, sessionStore: Store) => {
 
   const signOut = async () => {
     sessionStore.dispatch(logOut());
+    await localStore.remove(TEMPORARY_STORE_ID);
```

Comment:
`temporaryStore` should be overwritten properly when a user logs back in, but no harm in explicitly removing this to prevent any edge cases